Tests/AlicerceTests/AutoLayout/ConstraintGroupToggleTestCase.swift

```diff
@@ -0,0 +1,254 @@
+import XCTest
+@testable import Alicerce
+
+
+class ConstraintGroupToggleTestCase: BaseConstrainableProxyTestCase {
+
+    private enum TestConstraintGroupKey: Hashable {
+        case first
+        case second
+        case third
+    }
+
+    func testNoInitialConstraint_WithEnumKey_ShouldNotActivateConstraints() {
```

Comment:
format usually follows: `test<MethodOrPropertyName>_When<Conditions>_Should<ExpectedOutcome> {`

I think we're mixing the components a bit here 😅 
we're testing the `init`, and the conditions are e.g. "no initial constraint active". right?